app/src/main/java/ch/epfl/sdp/appart/user/AppUser.java

```diff
@@ -14,13 +14,20 @@
     private String profilePicture;
 
 
-    public AppUser(String userId, String name, String email, String password) {
-        if (userId == null || name == null || email == null) {
+    public AppUser(String userId, String name, String email, String phoneNumber, String profilePicture) {
```

Comment:
Actually I was thinking that the AppUser constructor could also simply be 

`public AppUser(String userId, String emai) { }`

This would be simpler since the actual user will almost always be initialised in a context where only these two attributes are known. At the moment I am implementing the User Profile UI (see #39) which will give the possibility to add and edit all information of the User. We could thus simply have `email` and `userId` in constructor and all the rest will be added by the User Profile UI (since it also could be that we find other **optional** attributes to add (age, gender, etc.) we can't then put them all in the constructor.